### PR TITLE
[MWPW-128064] Header/Footer component checks

### PR DESCRIPTION
**Pull request by chadsunvice**
* Header/Footer infrastructure
* Header first checks (a lot of the logic is under development on Milo side so these will be updated)

### CODE REVIEW

File: tests/feds/footer.test.js
Comment (amauch-adobe) [inline tests/feds/footer.test.js:1]: Perchance is this supposed to be empty? ;)

File: pages/feds.header.page.js
Comment (amauch-adobe) [inline pages/feds.header.page.js:1]: Looks like this file is also in another PR #90, just want to make sure they stay in sync.

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Footer.test.js is an empty file, don't know if it should be committed empty.  I also added the new tag labels you added in the two new spec files and added Nala to run against your PR.

File: pages/feds.header.page.js
Comment (chadsunvice) [reply pages/feds.header.page.js:1]: I've synced them beforehand to have no issue when merging. I have the latest version into the 3 PRs open.

File: tests/feds/footer.test.js
Comment (chadsunvice) [reply tests/feds/footer.test.js:1]: I can remove the file for now, I agree. The Footer isn't ready yet and this was a placeholder of sorts.

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**